internal/pkg/armeria/area.go

```diff
@@ -78,6 +78,12 @@ func (a *Area) MinimapJSON() string {
 			"x":     r.Coords.X(),
 			"y":     r.Coords.Y(),
 			"z":     r.Coords.Z(),
+			"north": r.Attribute("north"),
```

Comment:
Can you use the constants `AttributeNorth` (South, East, etc..) here instead of a string literal?